tests/test_get_replica_connections.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import subprocess
+from falkordb import FalkorDB
+import time
+
+
+# check=True is used to raise an exception if the command fails
+def delete_container(container_name):
+    subprocess.run(["docker","rm",container_name,'-f'],check=True,encoding='utf-8')
+# shell=True is used to run the command in a shell
+# encoding='utf-8' is used to get the output as a string
+def reapply_compose(path):
+    subprocess.run(f"docker-compose -f {path} down && docker-compose -f {path} up -d",check=True,shell=True,encoding='utf-8')
+
+def cluster_client():
+    client = FalkorDB(
+        host='localhost',
+        port=5000
+    )
+    return client
+
+
+def standalone_client():
+    client = FalkorDB(
+        host='localhost',
+        port=6379
+    )
+    return client
+
+def sentinel_client():
+    client = FalkorDB(
+        host='localhost',
+        port=26379
+    )
+    return client
+
+def delete_replicas(client,mode):
+    result = client.get_replica_connections()
+    if mode == 'cluster':
+        for i in result:
+            name = i[0]
+            delete_container(name)
+    elif mode == 'sentinel':
+        name = result[0][0]
+        delete_container(name)
+        
+
+def test_get_replica_connections_cluster():
+    c = cluster_client()
+    result = c.get_replica_connections()
+    assert ('node3',8000) in result
+    assert ('node4',9000) in result
+    assert ('node5',10000) in result
+    
+
+
+#
+def test_get_replica_connections_standalone():
+    c = standalone_client()
+    with pytest.raises(ValueError, match="Unsupported Redis mode"):
+        result = c.get_replica_connections()
+    
+
+#
+def test_get_replica_connections_sentinel():
+    c = sentinel_client()
+    result = c.get_replica_connections()
+    assert result == [('redis-server-2', 6381)]
+    
+
+
+
+def test_get_replica_connections_cluster_no_replicas():
+    # Assume this cluster has no replicas configured
+    delete_replicas(cluster_client(),'cluster')
```

Comment:
Instead of deleting the Docker container, use `SLAVEOF NOONE` to disconnect the slave from its master.
This should work on both Sentinel and Cluster.